Frontend/frontend/src/index.css

```diff
@@ -52,7 +52,7 @@
 body {
   font-family: "Nunito", sans-serif;
   min-height: 100vh;
-  background-color: var(--BGCOLOR);
+  /* background-color: var(--BGCOLOR); */
```

Comment:
Is there a reason why you comment this out?

Reply:
It's just the current color is is hard on the eyes, this is what it looks like now:
<img width="1703" alt="Screenshot 2023-10-30 at 21 34 41" src="https://github.com/mengzhuou/BizWeb/assets/81248183/37208ac9-ff36-4dea-b553-2d2ab6c68b25">
Not very important change could definately just remove that if needed